### PR TITLE
update serialization tutorial inline with changes to ksqlDB master

### PR DESCRIPTION
**Pull request by big-andy-coates**

### CODE REVIEW

**Comment by gAmUssA** [comment]
oh man @big-andy-coates, this will totally screw up my #375 :( 
Don't you mind if I incorporate those changes in my PR and close this?

**Comment by colinhicks** [comment]
@gAmUssA this PR is into the `ksqldb-latest` branch, where the latest ksqlDB version (effectively 0.10.0-SNAPSHOT) is being tested. #375 targets master and ksqlDB 0.9.0. I think it's best to have independent PRs for now. We can sort out the conflicts when it's time to merge `ksqldb-latest` into `master` at a later date.

**Comment by colinhicks** [comment]
@gAmUssA, we do use that Docker tag to internally validate the latest version of ksqlDB against existing tutorials. This allows us to incrementally keep the tutorials up to date with the latest ksqlDB release.

After a given ksqlDB release, as part of merging `ksqldb-latest` to master, we will change the images referenced in the tutorials to the released image, e.g. `ksqldb-server:0.10.0`. We would not publish changes to the live tutorials site which reference unreleased ksqlDB tags.

**Comment by gAmUssA** [comment]
@colinhicks 
tenor-99308718

Thank you

**Comment by big-andy-coates** [comment]
@colinhicks can I get an approval to merge this then?

File: _includes/tutorials/serialization/ksql/markup/dev/create-json-movies.adoc [hunk 1]
```diff
@@ -1,4 +1,5 @@
 The first thing we'll need is to create a Kafka topic and stream to represent the movie data. We declare the `VALUE_FORMAT` of the stream to be `json` to denote the format of the events.
+The stream stores the movie's id, `MOVIE_ID`, in the Kafka message's key, but could just as easily store it in a the value if we removed the `KEY` classifier.
```
Comment (colinhicks) [inline _includes/tutorials/serialization/ksql/markup/dev/create-json-movies.adoc:2]:
```suggestion
The stream stores the movie's id, `MOVIE_ID`, in the Kafka message's key, but could just as easily store it in the value if we removed the `KEY` classifier.
```

**Pull request merged**